cmd/stack.go

```diff
@@ -69,6 +70,13 @@ func newStackCmd() *cobra.Command {
 			if isCloud {
 				if cs, ok := s.(httpstate.Stack); ok {
 					fmt.Printf("    Owner: %s\n", cs.OrgName())
+					// If there is an in-flight operation, provide info.
+					if cs.CurrentOperation() != nil {
```

Comment:
To avoid the redundant `cs.CurrentOperation()`, why don't we declare it as a new variable. Such as:

```go
if currentOp := cs.CurrentOperation(); currentOp != nil {
    fmt.Printf("	Operation in progress: %s\n", currentOp.Kind)
}
```

IMHO, it is a lot easier to read that way.